scripts/process-resources.sh

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -p "haskellPackages.ghcWithPackages (pkgs: with pkgs; [split time])" -i runhaskell 
+
+import System.Environment 
+import Data.List.Split
+import Data.Bifunctor
+import Data.Time.Clock
+import Data.Time.Clock.POSIX
+
+main :: IO ()
+main = do
+  [file] <- getArgs
+  contents <- unzip 
+            . map (toTuple . splitOn ",") 
+            . lines 
+          <$> readFile file
+  let processedContents = bimap (map (posixSecondsToUTCTime . fromIntegral)) 
+                                (map toGB) 
+                                contents
+      finalResult = uncurry (zipWith (\a b -> show a ++ "," ++ show b)) 
+                            processedContents
+
+  putStrLn (unlines finalResult)
+
+toGB :: Int -> Int
```

Comment:
Technically - the SI unit of GiB is what you mean (GB are power of 10, GiB are powers of 2) see https://physics.nist.gov/cuu/Units/binary.html